### PR TITLE
Feature/swagger documentation

### PR DESCRIPTION
**Pull request by mhieta**
Downgrade project to use Django 2.2.9 LTS due to missing Django 3.x support by drf-yasg.
Add drf-yasg to project.
Update project to use django-enumfields.

### CODE REVIEW

File: traffic_control/models.py [hunk 1]
```diff
@@ -206,21 +216,21 @@ class TrafficSignPlan(models.Model):
...
-    color = models.IntegerField(_("Color"), choices=COLORS, default=BLUE)
+    color = EnumField(Color, verbose_name=_("Color"), default=Color.BLUE)
     road_name = models.CharField(_("Road name"), max_length=254, blank=True, null=True)
     lane_number = models.IntegerField(_("Lane number"), blank=True, null=True)
     lane_type = models.IntegerField(_("Lane type"), blank=True, null=True)
-    location_specifier = models.IntegerField(
-        _("Location specifier"),
-        choices=LOCATION_SPECIFIERS,
-        default=RIGHT,
+    location_specifier = EnumField(
```
Comment (ehaivala) [inline traffic_control/models.py]: Use `EnumIntegerField` with integer choices.

File: traffic_control/models.py [hunk 2]
```diff
@@ -313,33 +323,35 @@ class TrafficSignReal(models.Model):
...
-        choices=INSTALLATION_STATUSES,
-        default=ACTIVE,
+        default=InstallationStatus.ACTIVE,
     )
     installation_id = models.CharField(_("Installation id"), max_length=254)
     installation_details = models.CharField(
         _("Installation details"), max_length=254, blank=True, null=True
     )
-    condition = models.IntegerField(_("Condition"), choices=CONDITIONS, default=GOOD)
+    condition = EnumField(
```
Comment (ehaivala) [inline traffic_control/models.py]: Use `EnumIntegerField` with integer choices.

**Comment by ehaivala** [comment]
You might need to add a migration for the `django-enumfields` change.

File: traffic_control/models.py (hunk 1 above)
Comment (mhieta) [reply traffic_control/models.py]: Updated

File: traffic_control/models.py (hunk 2 above)
Comment (mhieta) [reply traffic_control/models.py]: Updated

**Pull request merged**